CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.16
+
+<!-- generated comparing v5.15.15..master -->
+
+_May 1, 2024_
+
+A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.15.16`
+
+- [material-ui][Dialog] Prevent onClick on the root element from being overwritten (@ryanburr) (#41914) @github-actions[bot]
+- [material-ui] Fix import statement in migration guide (@sai6855) (#41864) @github-actions[bot]
+- [material-ui][Select] Fix `muiName` property TypeScript error (@EyaOuenniche) (#41786) @github-actions[bot]
+- —>[material-ui][l10n] Fix typo in is-IS locale (@magnimarels) (#41815) @github-actions[bot]
+
+### Docs
+
+- Fix small SEO issues @oliviertassinari
+- Fix 301 Toolpad links @oliviertassinari
+- Fix 301 Toolpad links @oliviertassinari
+- Fix 301 image redirections @oliviertassinari
+- Fix small SEO issues @oliviertassinari
+- Fix 301 redirection @oliviertassinari
+- Fix format git diff regression (#41882) @oliviertassinari
+- Fix 301 links @oliviertassinari
+- [docs-infra] Fix analytics about inline ads (#41474) @alexfauquette
```

Comment:
This one should probably core within "Core" together with all other docs-infra entries?